From 39e430bfa33a84bb5a934767d8fb28f2d3f6c71c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 29 Oct 2020 22:22:37 +0200
Subject: [PATCH] [SW-0] cmd_module: add enable() documentation

Somehow the enable() callback documentation was missing.
Finally add it.

Change-Id: I92967fb3c1cd68ac064174ed5959462dda14db82
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/cmd_module.h b/common/cmd_module.h
index 6e66ed8..efa5e7c 100644
--- a/common/cmd_module.h
+++ b/common/cmd_module.h
@@ -40,6 +40,8 @@
  * run()		- run the handling of the command/request discovered by
  *			  the check() routine - can be slow since other modules
  *			  have been disabled by disable() routine
+ * enable()		- enable the module after it was disabled by disable()
+ *			  callback
  * run_fast()		- run the handling of the command/request discovered by
  *			  the check() routine - must take low amount of time
  *			  since other modules have not been disabled
